### PR TITLE
[frontend][onnx]fix pad constant value is none

### PR DESCRIPTION
**Pull request by chengven027-intellif**
hi, cc @AndrewZhaoLuo 
When I import my onnx model. I found my model get a error with pad. because the `pad` constant value is none. shown as follow:

It`s just a very little bug. So I fix it and no unnit test.

### CODE REVIEW

**Review by AndrewZhaoLuo — APPROVED** [review]

**Review by leandron — CHANGES_REQUESTED** [review]
Hi @chengven027-intellif, thanks for the PR. Any chance we can have a test case for this change?

**Comment by chengven027-intellif** [comment]
Dose this PR need to be reviewed again? @AndrewZhaoLuo @leandron

File: tests/python/frontend/onnx/test_forward.py [hunk 1]
```diff
@@ -2027,6 +2027,7 @@ def verify_pad_v11(indata, pads, mode="constant", value=0.0):
     verify_pad_v11(np.random.randn(2, 2).astype(np.float32), [0, 1, 0, 0], "constant", 0.0)
     verify_pad_v11(np.random.randn(2, 3).astype(np.float32), [1, 0, 0, 1], "constant", 0.0)
     verify_pad_v11(np.random.randn(3, 2).astype(np.float32), [0, 0, 1, 0], "constant", 5.0)
+    verify_pad_v11(np.random.randn(3, 2).astype(np.float32), [0, 0, 1, 0], "constant", False)
```
Comment (leandron) [inline tests/python/frontend/onnx/test_forward.py:2030]: Is setting `False` here (as opposed to `None`) reproducing the same issue this patch is fixing?

**Comment by AndrewZhaoLuo** [comment]
> Dose this PR need to be reviewed again? @AndrewZhaoLuo @leandron

Yeah I'll leave it to Leandron B)

File: tests/python/frontend/onnx/test_forward.py (hunk 1 above)
Comment (chengven027-intellif) [reply tests/python/frontend/onnx/test_forward.py:2030]: No, But I can't add a test case with `None`.
First, In the doc `test_forward.py   test_pad`, The `value` is a default number,can't set `None`.
`def verify_pad_v11(indata, pads, mode="constant", value=0.0)`
Then, the input node 'constant_value' is a `TensorProto.FLOAT`. not `None` too.
Can you give me some suggestions to add a test case with 'None'?  Thanks very much.

**Comment by chengven027-intellif** [comment]
I am sorry, I really can't create an unit test with empty string of `constant_value`.
@leandron @AndrewZhaoLuo

**Comment by AndrewZhaoLuo** [comment]
> I am sorry, I really can't create an unit test with empty string of `constant_value`. @leandron @AndrewZhaoLuo

In verify_pad_v11 I can seem to make the following:

```
          graph = helper.make_graph(
                [node],
                "pad_test",
                inputs=[
                    helper.make_tensor_value_info("input", TensorProto.FLOAT, list(indata.shape)),
                    helper.make_tensor_value_info("pads", TensorProto.INT64, (len(pads),)),
                    helper.make_tensor_value_info("constant_value", TensorProto.STRING, (1,)),
                ],
                initializer=[
                    helper.make_tensor("pads", TensorProto.INT64, (len(pads),), pads),
                    helper.make_tensor("constant_value", TensorProto.STRING, (1,), [b""]),
                ],
                outputs=[
                    helper.make_tensor_value_info("output", TensorProto.FLOAT, list(outdata.shape))
                ],
            )
```

However, my version of ONNXRT can't seem to accept the string type. What have you tried?

**Comment by chengven027-intellif** [comment]
Yes, I tried many times but I got the same error as yours.
`onnxruntime.capi.onnxruntime_pybind11_state.InvalidGraph: [ONNXRuntimeError] : 10 : INVALID_GRAPH : This is an invalid model. Type Error: Type 'tensor(string)' of input parameter (constant_value) of operator (Pad) in node () is invalid.`

**Comment by AndrewZhaoLuo** [comment]
Can you share your model by any chance? Can you also run your model via ONNX-RT?

**Comment by chengven027-intellif** [comment]
> Can you share your model by any chance? Can you also run your model via ONNX-RT?

[coat_model.zip](https://github.com/apache/tvm/files/9474424/coat_model.zip)

1. model have been uploaded.
2. The model can pass by ONNX-RT. And the floating point comparison between tvm and onnx-RT result can also pass.

**Comment by AndrewZhaoLuo** [comment]
Sorry, I'll take a look tomorrow.

**Comment by chengven027-intellif** [comment]
Hey, @AndrewZhaoLuo  Have you seen this PR, and any update with me? thanks.

**Comment by AndrewZhaoLuo** [comment]
Oops, I am so sorry, I have been very busy lately. I will try to get to this this weekend
